rita/src/exit.rs

```diff
@@ -176,6 +178,7 @@ fn main() {
     assert!(rita_common::tunnel_manager::TunnelManager::from_registry().connected());
     assert!(rita_common::http_client::HTTPClient::from_registry().connected());
     assert!(rita_common::traffic_watcher::TrafficWatcher::from_registry().connected());
+    assert!(PeerListener::new().is_ok());
```

Comment:
What does this do? Looks like it just creates a peer listener, check if it succeeds and then throws it away?

Reply:
previously new did a lot, now we need the same stuff as the other actors I guess.